src/schema/v2/image/resized.ts

```diff
@@ -59,6 +75,28 @@ const ResizedImageUrlType = new GraphQLObjectType<any, ResolverContext>({
     },
     url: {
       type: GraphQLString,
+      resolve: ({ src, resize: { width, height } }) =>
+        proxy(src, "resize", width, height),
+    },
+    src: {
+      type: GraphQLString,
+      resolve: ({ src, resize: { width, height } }) =>
+        proxy(src, "resize", width, height),
+    },
+    srcSet: {
+      type: GraphQLString,
+      description:
+        "Returns a `srcSet` string with 1x, 2x, 3x pixel densities based on passed in arguments",
+      resolve: ({ src, resize: { width, height } }) =>
+        SUPPORTED_PIXEL_DENSITIES.map(({ density, quality }) =>
+          proxy(
+            src,
+            "resize",
+            width && width * density,
+            height && height * density,
+            quality
+          )
+        ).join(", "),
```

Comment:
```gql
{
  artist(id: "gerhard-richter") {
    id
    image {
      width
      resized(height: 200) {
        width
        height
        src
        srcSet
      }
    }
  }
}
```

```json
{
  "data": {
    "artist": {
      "id": "QXJ0aXN0OjRkZjY5MzQ5YmMzY2YxMDAwMTAwMDcyNg==",
      "image": {
        "width": 711,
        "resized": {
          "width": 138,
          "height": 200,
          "src": "https://d7hftxdivxxvm.cloudfront.net?resize_to=fit&width=138&height=200&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2F8yqS4Lx1H9kfjDhDpk1Q4w%2Flarge.jpg",
          "srcSet": "https://d7hftxdivxxvm.cloudfront.net?resize_to=fit&width=138&height=200&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2F8yqS4Lx1H9kfjDhDpk1Q4w%2Flarge.jpg, https://d7hftxdivxxvm.cloudfront.net?resize_to=fit&width=276&height=400&quality=65&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2F8yqS4Lx1H9kfjDhDpk1Q4w%2Flarge.jpg, https://d7hftxdivxxvm.cloudfront.net?resize_to=fit&width=414&height=600&quality=50&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2F8yqS4Lx1H9kfjDhDpk1Q4w%2Flarge.jpg"
        }
      }
    }
  }
}
```

Reply:
#superminor - nested implicit returns make things hard to debug later through, sinice one can only easily attach breakpoints to concrete statements such as variables or `return`; otherwise everything is evaluated inline and the debugger goes weird.